plugins/woocommerce-blocks/tests/e2e/tests/templates/legacy-templates.block_theme.spec.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+import { cli } from '@woocommerce/e2e-utils';
+
+test.describe( 'Legacy templates', async () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllTemplates( 'wp_template' );
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllTemplates( 'wp_template' );
+	} );
+
+	test( 'woocommerce//* name is supported', async ( {
```

Comment:
Nit, should we call it `slug` instead of `name`? This is how we refer to it in the code.

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/46269/commits/54b35bf2ab3e0173fb0b9bdb9deb1acb00c48dc2